scarb/tests/build_cairo_plugin.rs

```diff
@@ -1431,3 +1431,175 @@ fn can_be_expanded() {
         expanded,
     );
 }
+
+#[test]
+fn can_expand_trait_inner_func_attrr() {
+    let temp = TempDir::new().unwrap();
+    let t = temp.child("some");
+    CairoPluginProjectBuilder::default()
+        .lib_rs(indoc! {r##"
+            use cairo_lang_macro::{attribute_macro, ProcMacroResult, TokenStream};
+
+            #[attribute_macro]
+            pub fn some(_attr: TokenStream, token_stream: TokenStream) -> ProcMacroResult {
+                ProcMacroResult::new(TokenStream::new(
+                    token_stream.to_string()
+                    .replace("hello", "world")
+                    .replace("12", "34")
```

Comment:
nit: we replace literals here, it's also good to test whether they were replaced after running the changed function. Similar way as it's done for impl test below.